PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# Description
+
+<Info on what this pull request adds>
+
+[//]: # (Reminder: Always update the "lastmod" portion at the top of any pages you edit)
```

Comment:
Can we get `at the top of any pages you edit` changed to `at the top of any modified Markdown files` please?